features/admin.extensions.v1/components/administrators/wizard/steps/admin-user-basic.tsx

```diff
@@ -158,7 +158,8 @@ export const AddAdminUserBasic: React.FunctionComponent<AddAdminUserBasicProps>
                 .then((response: AxiosResponse) => {
                     setRolesList(response.data.Resources);
                     response.data.Resources.map((role: RolesInterface, index: number) => {
-                        if (role.displayName !== "system" &&
+                        if ((!legacyAuthzRuntime || role.meta?.systemRole) &&
```

Comment:
Wouldn't this condition show the non-system roles in new authz runtime? is it expected?

Reply:
yes this is expected

Reply:
Let's use the config defined in https://github.com/wso2/identity-apps/blob/master/apps/console/src/public/deployment.config.json#L1154 also, to conditionally enable system roles depending on the deployment.